pkg/manager/manager.go

```diff
@@ -1890,11 +1890,69 @@ func (m *jobManager) CheckValidJobConfiguration(req *JobRequest) error {
 	return nil
 }
 
+// validVersionRegexes represents all the valid version formats we recognize for OpenShift.
+// We allow an optional comma in the front so we can account for both 4.19,xxxx and xxxx,4.19
+var validVersionRegexes = []*regexp.Regexp{
+	regexp.MustCompile(`^\d+\.\d+`),               // 4.19
+	regexp.MustCompile(`^nightly`),                // nightly
+	regexp.MustCompile(`^ci`),                     // ci
+	regexp.MustCompile(`^\d+\.\d+\.nightly`),      // 4.19.nightly
+	regexp.MustCompile(`^\d+\.\d+\.ci`),           // 4.19.ci
+	regexp.MustCompile(`^\d+\.\d+\.0-0\.nightly`), // 4.19.0-0.nightly
+	regexp.MustCompile(`^\d+\.\d+\.0-0\.ci`),      // 4.19.0-0.ci
+
+	// Release specs for nightly and ci
+	regexp.MustCompile(`^.*release:\d+\.\d+\.\d+-0\.nightly-\d{4}-\d{2}-\d{2}-\d{6}$`),
+	regexp.MustCompile(`^.*release:\d+\.\d+\.\d+-0\.ci-\d{4}-\d{2}-\d{2}-\d{6}$`),
+
+	// Quay.io pullspecs
+	regexp.MustCompile(`^quay\.io/openshift-release-dev/.+:(\d+\.\d+\.\d+(-ec\.\d+)?|.+-0\.(nightly|ci)(-priv)?-\d{4}-\d{2}-\d{2}-\d{6})$`),
+
+	// quay.io/openshift-release-dev/ocp-release:4.19.0-ec.4-x86_64
+	regexp.MustCompile(`^quay\.io/openshift-release-dev/ocp-release:\d+\.\d+\.\d+-ec\.\d+(-\w+)?$`),
+
+	// quay.io/okd/scos-release:4.19.0-okd-scos.ec.8
+	regexp.MustCompile(`^quay\.io/okd/scos-release:\d+\.\d+\.\d+-okd-scos\.ec\.\d+$`),
+
+	// OKD versions
+	regexp.MustCompile(`^(quay\.io/okd|quay\.io/openshift/okd|registry\.ci\.openshift\.org/origin/release-scos):\d+\.\d+\.\d+-0\.okd(-scos)?(\.ec\.\d+)?(-\d{4}-\d{2}-\d{2}-\d{6})?$`),
```

Comment:
Where does the need for `quay\.io/openshift/okd` come from?  I couldn't find anything in the release-controller page that references it...
https://amd64.origin.releases.ci.openshift.org/

Reply:
that was mentioned in [here](https://github.com/openshift/ci-chat-bot/pull/553#discussion_r2052869734) but I can take it out if it's not necessary.